src/components/panel/Panel.js

```diff
@@ -61,4 +63,11 @@ export default class PanelComponent extends NestedComponent {
       }
     });
   }
+
+  getComponent(path, fn, originalPath) {
+    if (this.parent instanceof Wizard && this.parent.parent instanceof FormComponent) {
```

Comment:
What if the Panel is inside of another panel and then inside of the form?... wouldn't this break?  I believe the following would work.

```
if (this.root.parent && this.root.parent instanceof FormComponent) {
  ...
}
```

Reply:
Yes, that makes sense.